03/Items.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+
+function Items(props) {
+    const {data, inCart, clickHandle} = props;
+
+    function handleClick(id) {
+        if(typeof clickHandle === 'function') {
+            clickHandle(id);
+        }
+    }
+
+
+    return (
+        <li>{data.id} {data.name} {(data.price)} <button disabled={inCart ? true :false} onClick={() => handleClick(data.id)}>Kup Teraz</button></li>
```

Comment:
Po oddaniu do koszyka należałoby zmienić tekst z "kup teraz" na "usuń z koszyka" i faktycznie obsłużyć to usuwanie :)